src/v/config/configuration.cc

```diff
@@ -92,24 +114,26 @@ configuration::configuration()
        .visibility = visibility::user},
       std::chrono::weeks{2},
       {.min = 60s})
-  , log_segment_ms_min(
+  , log_segment_ms_min(*this, "log_segment_ms_min")
+  , log_segment_ms_max(*this, "log_segment_ms_max")
+  , log_segment_ms_locked_min(
       *this,
-      "log_segment_ms_min",
-      "Lower bound on topic segment.ms: lower values will be clamped to this "
-      "value",
+      "log_segment_ms_locked_min",
+      "Lower bound on topic segment.ms; Reject topics with lower values",
       {.needs_restart = needs_restart::no,
        .example = "60000",
-       .visibility = visibility::tunable},
-      60s)
-  , log_segment_ms_max(
+       .visibility = visibility::user},
+      60s,
+      validate_locked_min<std::chrono::milliseconds>)
```

Comment:
https://github.com/redpanda-data/redpanda/issues/13468#issuecomment-1729476182

Reply:
responded in the ticket.